src/main/java/core/basesyntax/FigureDrawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDrawable {
+    void getInfo();
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. However, in this case, since there's only one method related to drawing, it's fine.